data/prepro.py

```diff
@@ -189,20 +189,20 @@ def get_image_ids(data, id2path):
 
     if args.download:
         if args.version == '1.0':
-            os.system('wget https://www.dropbox.com/s/ix8keeudqrd8hn8/visdial_1.0_train.zip?dl=0')
-            os.system('wget https://www.dropbox.com/s/ibs3a0zhw74zisc/visdial_1.0_val.zip?dl=0')
+            os.system('wget -O visdial_1.0_train.zip https://www.dropbox.com/s/ix8keeudqrd8hn8/visdial_1.0_train.zip?dl=0')
+            os.system('wget -O visdial_1.0_val.zip https://www.dropbox.com/s/ibs3a0zhw74zisc/visdial_1.0_val.zip?dl=0')
```

Comment:
Could also just trim the trailing `?dl=0` from download links, but this works too.

Reply:
Done :+1: